api/authFunctions.ts

```diff
@@ -0,0 +1,22 @@
+import { account } from "./config"
+import { NextRouter } from 'next/router';
+
+
+  export const login = async (router: NextRouter, email: string, password: string) => {
```

Comment:
Why are we passing router as prop? Should this not just be a global const in this file?

Reply:
Initially, I wanted to keep it as a global const but ESLint underlined it red as `React Hook "useRouter" cannot be called at the top level. React Hooks must be called in a React function component or a custom React Hook function.`

I do not want to convert this into a function component and I pictured this file as a util file for auth functions to live. 

After searching around and asking ChatGPT for suggestions, it mentioned I could pass router, but honestly, I'm not a fan of this either. I will keep looking around for suggestions.

Please let me know if you have any suggestions/workaround for this.